docs/node-operators/get-started/setup-datanode.md

```diff
@@ -326,18 +332,149 @@ vega datanode start --home="YOUR_DATA_NODE_HOME_PATH"
+vega datanode network-history fetch <segment-id-of-segment-at-height-X> 2000 --home="YOUR_DATA_NODE_HOME_PATH"
+```
+
+Once the network history segments have been downloaded, running: 
+
+```shell
+vega datanode network-history show --home="YOUR_DATA_NODE_HOME_PATH"
+``` 
+
+should display the network history you have:
+
+```text
+Available contiguous history spans:
+
+Contiguous history from block height XXXXX to XXXXX, from segment id: <first-segment-id> to <last-segment-id>
+
+
+Datanode contains no data
+```
+
+### Loading network history into Data Node
+
+Now that you have downloaded historical data you can load the history into the data node using:
+
+```shell
+vega datanode network-history load --home="YOUR_DATA_NODE_HOME_PATH"
+```
+
+:::note
+Data node must not be running when you use this command. If you attempt to use `network-history load` while data node is running, you will get an error:
+
+```text
+datanode must be shutdown before data can be loaded
+```
+:::
+
+You will be notified that the load command will force all existing connections to the data node database will be closed automatically and prompted if you want to continue. Enter `y` and data node will check how much history you have, tell you what it will load, and prompts you to confirm if you want to continue. Enter `y` again and the data node will load the history you have fetched.
+
+This process may take a very long time depending on how much history you have retrieved and are loading into the database. Some processes in the restoration can take a long time to complete and it may look like the process is not doing anything. Do not try to terminate the process early or you may end up with a corrupted data node database.
+
+
+## Data Node recovery from Network History
+
+It is possible to start a data node and initialize it using network history automatically. By default, this process is disabled, and if enabled will only fetch - if you don't have the snapshot locally - the last segment before loading it into the data node. This is to allow users to quickly initialize a data node without fetching a lot of data and start participating while the data node is fetching more data in the background.
+
+To enable this feature set the `AutoInitialiseFromNetworkHistory` setting in the data node configuration file to `true`. It is safe to leave this setting to true. If `WipeOnStartup` is `true`, data node will always restart with the last network history it is able to retrieve. If `WipeOnStartup` is `false`, data node will not reload data if the block height available in the database is equal to or greater than the block height from the network history.
+
+If you want to fetch more than the last segment, you may also set the `MinimumBlockCount` configuration setting.
+
+If you want to initialise the data node automatically up to a specific segment and X blocks before it, you can set both the `ToSegment` and `MinimumBlockCount` configuration settings.
```

Comment:
Can We add information here on what timeouts We must update?

Reply:
Resolved https://github.com/vegaprotocol/documentation/pull/556/commits/7c42130487b2326cabd0e9470ba7f297376f173a